solutions/loading-web-fonts/pages/index.tsx

```diff
@@ -0,0 +1,213 @@
+          pellentesque. Praesent tincidunt elementum tempus.
+        </FontMixer>
+        <Text>
+          Fonts with different sizes not only may create a layout shift in its
+          space but could also push or pull content around it creating a bigger
+          shift in the UI.
+        </Text>
+        <Text>
+          One possible solution is having an adjusted fallback font that tries
+          to match the used space of our custom font to reduce the layout shift.
+          You can use{' '}
+          <Link href="https://www.industrialempathy.com/perfect-ish-font-fallback/?font=Inter">
+            this tool
+          </Link>{' '}
+          to find which fallback font might match your custom font.
+        </Text>
+        <Text>
+          Here we can see the difference that using an adjusted fallback font
+          makes:
+        </Text>
+        <FontMixer fonts={['Inter', 'Inter-fallback']}>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas
+          convallis mauris dui, non placerat urna fringilla at. Suspendisse a
+          orci quis arcu tristique sagittis at sed leo. Nam quis neque dapibus,
+          semper nisl non, lacinia ligula. Duis viverra a nisl ut consectetur.
+          Ut elementum fringilla odio viverra egestas. Morbi aliquet lorem
+          lorem, in suscipit nibh tempus quis. Mauris gravida dapibus odio,
+          vitae interdum lectus pretium ut. Vivamus tincidunt laoreet
+          pellentesque. Praesent tincidunt elementum tempus.
+        </FontMixer>
+      </section>
+
+      <section className="flex flex-col gap-3">
+        <Text variant="h2">Font display</Text>
+        <Text>
+          As we mentioned above, during the timespan between our page displaying
+          text before loading our custom font and our custom font being applied
+          to that text, we can use several strategies that will determine how
+          that text will be displayed. For changing this behaviour we can use
+          the CSS property <Code>font-display</Code>.
+        </Text>
+        <List>
+          <li>
+            <Image src={displayAuto} alt="font-display: auto" />
+            <Code>font-display: auto</Code>: This is the default setting for{' '}
+            <Code>font-display</Code>. It will make the text invisible while the
+            custom is being loaded, the space reserved depends on the space that
+            the fallback font configured would have taken to fill show that
+            text.
+          </li>
+          <li>
+            <Image src={displaySwap} alt="font-display: swap" />
+            <Code>font-display: swap</Code>: Will display the text using the
+            fallback font until our custom font is loaded and will then swap the
+            fallback font with the custom font.
+          </li>
+          <li>
+            <Image src={displayOptional} alt="font-display: optional" />
```

Comment:
There is the nuance that if you do not have a preload for the font (which triggers a special heuristic) then you will never see the font on first page load in a session, even if it is in disk cache (you'd only see it if it is in memory cache)